testing/architecture-test/src/test/java/org/gradle/architecture/test/KotlinCompatibilityTest.java

```diff
@@ -78,7 +80,7 @@ private static class Accessor {
         private final JavaMethod method;
         private final boolean isGetter;
 
-        @javax.annotation.Nullable
+        @org.jspecify.annotations.Nullable
```

Comment:
Is there a reason these aren't using an `import` now?

Reply:
Good catch, fixed!